.github/workflows/test-only.yml

```diff
@@ -50,10 +50,12 @@ jobs:
       - name: Test built container with pytest
         run: |
           # Unit tests
+          echo "run test with unittest"
```

Comment:
![image](https://github.com/dgtlmoon/changedetection.io/assets/61624808/180aaa82-4af1-469d-ad26-c0bf1d7333ac)

I couldn't find the way to suppress logs "loguru, flask and unittest" combination.
To clarify the problem, I added messages.